cfgov/v1/models/organisms.py

```diff
@@ -22,6 +22,22 @@ class Meta:
         template = '_includes/organisms/full-width-text.html'
 
 
+class ImageText5050Group(blocks.StructBlock):
+    image_texts = blocks.ListBlock(molecules.ImageText5050())
```

Comment:
It's not necessary for the Image Text 25-75 because it goes full-width, but the 50-50 and link blob do a half-layout thing.